benchmark/multi_gpu/training/training_benchmark_xpu.py

```diff
@@ -48,6 +48,11 @@ def custom_optimizer(model: Any, optimizer: Any) -> Tuple[Any, Any]:
 
     assert args.dataset in supported_sets.keys(), \
         f"Dataset {args.dataset} isn't supported."
-    data, num_classes = get_dataset(args.dataset, args.root)
+
+    if rank == 0:
```

Comment:
nit: Please add a comment explaining why we need this, e.g. "if the dataset is not present, it will be downloaded only by process rank=0".

Reply:
LGTM